packages/cysync-core/src/pages/OnBoarding/OnboardingPageLayout.tsx

```diff
@@ -83,7 +82,7 @@ export const OnboardingPageLayout: React.FC<OnboardingPageLayoutProps> = ({
       title={title}
       subTitle={subTitle}
       version={`ver ${window.cysyncEnv.VERSION}`}
-      headerProps={parseHeaderProps({ withHelp, withEmail, lang })}
+      headerProps={parseHeaderProps()}
       footerProps={parseFooterProps({ withBack, lang })}
```

Comment:
There is an inconsistency between `parseHeaderProps` and `parseFooterProps`. If it is a necessary change, then please add a comment, otherwise try making it consistent.